config/routes.rb

```diff
@@ -71,12 +73,14 @@
     resource :job, only: [] do
       get :application_details, to: 'hiring_staff/vacancies/application_details#new'
       post :application_details, to: 'hiring_staff/vacancies/application_details#create'
+      get :supporting_documents, to: 'hiring_staff/vacancies/supporting_documents#new'
+      post :supporting_documents, to: 'hiring_staff/vacancies/supporting_documents#create'
       get :candidate_specification, to: 'hiring_staff/vacancies/candidate_specification#new'
       post :candidate_specification, to: 'hiring_staff/vacancies/candidate_specification#create'
       get :documents,
         to: 'hiring_staff/vacancies/documents#index',
         constraints: lambda {
-          |request| ENV['FEATURE_UPLOAD_DOCUMENTS'] == true
+          |request| ENV['FEATURE_UPLOAD_DOCUMENTS'] == 'true'
```

Comment:
What's the reason `true` was changed from boolean to string? Is it better as a string? Do ENV variables need to be strings?

Reply:
This will eventually bite as not everything casts it as a string.  

https://github.com/DFE-Digital/teacher-vacancy-service/commit/a4cd13097df5d148e59e63407852d22f127a101c

Reply:
This has been changed to:
```
|request| UploadDocumentsFeature.enabled?
```